### PR TITLE
Reset image links to master

### PR DESCRIPTION
**Pull request by syedjawadakhtar**
Images were linked to my patch. Changed to master branch.

### CODE REVIEW

File: docs/computer/Acoustics.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ Points would be awarded as follows. If the AUV successfully causes the ball to d
 flare with the pinger: 40 Points
 [This task from the [rulebook of SAUVC 2018](https://sauvc.org/2018/)]
 
-![yellow_Flare_animated](https://github.com/auvzhcet/Documentation/tree/jawad-patch-1/docs/computer/static/flare_Animation.png)
-![yellow_Flare_underwater](https://github.com/auvzhcet/Documentation/tree/jawad-patch-1/docs/computer/static/yellow_Flare_water.png)
+![yellow_Flare_animated](https://github.com/auvzhcet/Documentation/tree/master/docs/computer/static/flare_Animation.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: This is not the correct way to add images. You are actually pointing GitHub link. Do it like this:
Flare animation

File: docs/computer/Acoustics.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ Points would be awarded as follows. If the AUV successfully causes the ball to d
...
+![yellow_Flare_underwater](https://github.com/auvzhcet/Documentation/tree/master/docs/computer/static/yellow_Flare_water.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: Update this!

File: docs/computer/Acoustics.md [hunk 3]
```diff
@@ -39,9 +39,9 @@ Drop the ball in the blue drum: 30 Points
 Drop the ball in the red drum with the pinger: 50 Points
 Dropping the ball in any other red drum: 10 Points
 
-![drums](https://github.com/auvzhcet/Documentation/tree/jawad-patch-1/docs/computer/static/drums.png)
+![drums](https://github.com/auvzhcet/Documentation/tree/master/docs/computer/static/drums.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: This also.

File: docs/computer/Acoustics.md [hunk 4]
```diff
@@ -39,9 +39,9 @@ Drop the ball in the blue drum: 30 Points
...
 
-![drums_top_view](https://github.com/auvzhcet/Documentation/tree/jawad-patch-1/docs/computer/static/drums_top_view.png)
+![drums_top_view](https://github.com/auvzhcet/Documentation/tree/master/docs/computer/static/drums_top_view.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: This also!

File: docs/computer/Acoustics.md [hunk 5]
```diff
@@ -63,21 +63,21 @@ Key features:
 3.Inbuilt IMU that can be used further for movement of the vehicle.
 4.WiFi connectivity.
 
-![NI myRIO-1900 Hardware Block Diagram](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRIO_hardware_pinout.png)
+![NI myRIO-1900 Hardware Block Diagram](https://github.com/auvzhcet/Documentation/blob/master/docs/computer/static/myRIO_hardware_pinout.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: This one also!

File: docs/computer/Acoustics.md [hunk 6]
```diff
@@ -63,21 +63,21 @@ Key features:
...
 
-![Primary/Secondary Signals on MXP Connectors A and B](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRio_ports.png)
-_Source: [NI MyRIO User Guide and Specification](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/NI_MyRIO_User_Guide_and_Specification.pdf)_
+![Primary/Secondary Signals on MXP Connectors A and B](https://github.com/auvzhcet/Documentation/blob/master/docs/computer/static/myRio_ports.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: =>

File: docs/computer/Acoustics.md [hunk 7]
```diff
@@ -63,21 +63,21 @@ Key features:
...
+_Source: [NI MyRIO User Guide and Specification](https://github.com/auvzhcet/Documentation/blob/master/docs/computer/static/NI_MyRIO_User_Guide_and_Specification.pdf)_
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: This I am not sure!
Got to check this. Try to point this also in the same way that I suggested and make sure you send the patch after testing locally.

File: docs/computer/Acoustics.md [hunk 8]
```diff
@@ -63,21 +63,21 @@ Key features:
...
 
 ## The process
 
 ### Placement of hydrophones  
 
 They are placed at specific distances in an equilateral shape for accurate results.
 Reference: [Design and analysis of air acoustic vector-sensor configurations for two-dimensional
-geometry](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/wajid2016.pdf)
+geometry](https://github.com/auvzhcet/Documentation/blob/master/docs/computer/static/wajid2016.pdf)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md:77]: Test this also!

Comment (pnijhara) [reply docs/computer/Acoustics.md:77]: Though this can work.

File: docs/computer/Acoustics.md [hunk 9]
```diff
@@ -63,21 +63,21 @@ Key features:
...
 The approximate distance of the placement of the arrays is calculated to be 3 inches apart. The distance is measured from the center of the hydrophones. It has to be in the form of an equilateral triangle, as seen in the figure.
 
-![Star configuration, having four measured signals, i.e., _pc_, _pk_ , _pl_ , and _pm_.](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/microphone_placement.png)
+![Star configuration, having four measured signals, i.e., _pc_, _pk_ , _pl_ , and _pm_.](https://github.com/auvzhcet/Documentation/blob/master/docs/computer/static/microphone_placement.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md]: Make this change also!

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: docs/computer/Acoustics.md [hunk 10]
```diff
@@ -63,21 +63,21 @@ Key features:
 3.Inbuilt IMU that can be used further for movement of the vehicle.
 4.WiFi connectivity.
 
-![NI myRIO-1900 Hardware Block Diagram](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRIO_hardware_pinout.png)
+![NI myRIO-1900 Hardware Block Diagram](static/myRIO_hardware_pinout.png)
 
-![Primary/Secondary Signals on MXP Connectors A and B](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/myRio_ports.png)
-_Source: [NI MyRIO User Guide and Specification](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/NI_MyRIO_User_Guide_and_Specification.pdf)_
+![Primary/Secondary Signals on MXP Connectors A and B](static/myRio_ports.png)
+_Source: [NI MyRIO User Guide and Specification](static/NI_MyRIO_User_Guide_and_Specification.pdf)_
```
Comment (pnijhara) [inline docs/computer/Acoustics.md:69]: Is this working fine locally?

Comment (pnijhara) [reply docs/computer/Acoustics.md:69]: If yes! then please change other pdf links also. Else, let other pdf links to be like that github link.

File: docs/computer/Acoustics.md [hunk 11]
```diff
@@ -63,21 +63,21 @@ Key features:
...
 ### Placement of hydrophones  
 
 They are placed at specific distances in an equilateral shape for accurate results.
 Reference: [Design and analysis of air acoustic vector-sensor configurations for two-dimensional
-geometry](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/wajid2016.pdf)
+geometry](https://github.com/auvzhcet/Documentation/blob/master/docs/computer/static/wajid2016.pdf)
 The approximate distance of the placement of the arrays is calculated to be 3 inches apart. The distance is measured from the center of the hydrophones. It has to be in the form of an equilateral triangle, as seen in the figure.
 
-![Star configuration, having four measured signals, i.e., _pc_, _pk_ , _pl_ , and _pm_.](https://github.com/auvzhcet/Documentation/blob/jawad-patch-1/docs/computer/static/microphone_placement.png)
+![Star configuration, having four measured signals, i.e., _pc_, _pk_ , _pl_ , and _pm_.](static/microphone_placement.png)
```
Comment (pnijhara) [inline docs/computer/Acoustics.md:80]: Is the written part important to show up? If yes, then you got to write it down again after image.

**Review by pnijhara — CHANGES_REQUESTED** [review]

File: docs/computer/Acoustics.md (hunk 11 above)
Comment (syedjawadakhtar) [reply docs/computer/Acoustics.md:80]: I'll add captions to it and remove the written part.

Comment (pnijhara) [reply docs/computer/Acoustics.md:80]: Sounds Good!
Send another patch for this.

**Pull request merged**